nostrdb/flatcc/flatcc_builder.h

```diff
@@ -82,7 +82,7 @@ extern "C" {
  * Note: some internal assertion will remain if disabled.
  */
 #ifndef FLATCC_BUILDER_ASSERT_ON_ERROR
-#define FLATCC_BUILDER_ASSERT_ON_ERROR 1
+#define FLATCC_BUILDER_ASSERT_ON_ERROR 0
```

Comment:
Have we ruled out the risk that the aforementioned errors could corrupt flatbuffer data and cause further issues down the line?

Reply:
it is handled here:

```
	NdbProfile_parse_json_table(ctx, buf, buf + bufsiz, profile);
	if (ctx->error)
		return 0;
```

in `ndbprofile_parse_json`

Reply:
Nice!

Do we have similar guards around other flatbuffer objects like ndbnotes and such (given that this constant looks global)? If we do, then this LGTM